content/en/docs/concepts/architecture/nodes.md

```diff
@@ -26,8 +26,8 @@ architecture design doc for more details.
 A node's status contains the following information:
 
 * [Addresses](#addresses)
-* [Condition](#condition)
-* [Capacity](#capacity)
+* [Conditions](#conditions)
```

Comment:
This change will break inbound hyperlinks.
Instead, you could combine
```suggestion
 * [Conditions](#condition)
```

with the suggestion below.